core/src/main/kotlin/model/ContentSourceSet.kt

```diff
@@ -0,0 +1,32 @@
+package org.jetbrains.dokka.model
+
+import org.jetbrains.dokka.DokkaConfiguration.DokkaSourceSet
+import org.jetbrains.dokka.DokkaSourceSetID
+import org.jetbrains.dokka.Platform
+
+data class ContentSourceSet(
```

Comment:
Naming alternatives welcome. I am not 100% happy with this name :/

Reply:
Alternaives: 
- `DisplaySourceSet`  -> "Because this is what is displayed
- `DisplayableSourceSet` ->".."
- `SourceSetBubble`-> "Because in the end, it represents those bubbles"
- `MergedSourceSet` -> Because it can represent multiple original source sets
- `CompositeSourceSet`-> ...
... 

Please, help 🙈

Reply:
`CompositeSourceSet` would be most consistent with `ContentSourceSetId`. But on the other hand the most important thing in this class is not that it is any composite of source sets, but composite that can be displayed in the final documentation. I would go with `DisplayableSourceSet` / `PresentationSourceSet`.